Tasks/SonarQubePreBuild/SonarQubePreBuild.ps1

```diff
@@ -13,51 +13,34 @@ param(
 
 Write-Verbose "Starting SonarQube Pre-Build Setup Step"
 
-Write-Verbose -Verbose "connectedServiceName = $connectedServiceName"
-Write-Verbose -Verbose "projectKey = $projectKey"
```

Comment:
I think without -verbose option, you'd need the build variable 'system.debug' set to 'true' for these messages to show up in output..